src/BPEssentials/Modules/ModuleHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using BPCoreLib.Util;
+using BrokeProtocol.API;
+using System.Collections.Generic;
+
+namespace BPEssentials.Modules
+{
+    public class ModuleHandler
+    {
+        public Logger Logger { get; set; }
+
+        public List<Module> Modules { get; } = new List<Module>();
+
+        public void Initialize()
+        {
+            FindAll();
+        }
+
+        public void FindAll()
+        {
+            foreach (var plugin in (IEnumerable<Plugin>)BPAPI.Instance.Plugins)
```

Comment:
I believe this cast is not required.
```suggestion
            foreach (var plugin in BPAPI.Instance.Plugins)
```